src/main/java/com/minecolonies/coremod/entity/pathfinding/WalkToProxy.java

```diff
@@ -237,42 +239,43 @@ else if(targetY <= levelDepth)
 
     /**
      * Returns a proxy point to the goal.
-     * @param target the target.
+     *
+     * @param target         the target.
      * @param distanceToPath the total distance.
      * @return a proxy or, if not applicable null.
      */
     @NotNull
     private BlockPos getProxy(@NotNull BlockPos target, @NotNull BlockPos position, double distanceToPath)
     {
-        if(worker.getColony() == null)
+        if (worker.getColony() == null)
         {
             return target;
         }
 
         double weight = Double.MAX_VALUE;
         BlockPos proxyPoint = null;
 
-        for(BlockPos wayPoint: worker.getColony().getWayPoints(position, target))
+        for (BlockPos wayPoint : worker.getColony().getWayPoints(position, target))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'wayPoint' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)